FirebaseAdmin/FirebaseAdmin/Auth/Hash/Md5.cs

```diff
@@ -24,17 +24,17 @@ public sealed class Md5 : RepeatableHash
         /// Initializes a new instance of the <see cref="Md5"/> class.
         /// Defines the name of the hash to be equal to MD5.
         /// </summary>
-        public Md5()
+        internal Md5()
```

Comment:
public

Anything that we expect the developers to instantiate must be public. Only make the abstract base classes like `RepeatableHash` internal (or rather make their constructors internal).